great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -15,10 +17,10 @@
 from great_expectations.core import IDDict
 from great_expectations.core.batch import BatchMarkers, BatchSpec
 from great_expectations.core.batch_spec import (
+    RuntimeDataBatchSpec,
```

Comment:
This does not appear to be used

Reply:
Ah - it's coming from Develop but it is not used there either

Reply:
@anthonyburdi Ditto from the above! :)